plugins/services/src/js/containers/service-connection/ServiceConnectionEndpointList.js

```diff
@@ -134,12 +134,6 @@ class ServiceConnectionEndpointList extends React.Component {
             {this.getHostPortValue(portDefinition, service)}
           </ConfigurationMapValue>
         </ConfigurationMapRow>
-        <ConfigurationMapRow key="service-port">
-          <Trans render={<ConfigurationMapLabel />}>Service Port</Trans>
-          <ConfigurationMapValue>
-            {this.getServicePortValue(portDefinition)}
```

Comment:
i'd love if we removed `getServicePortValue` as this was the only place we used it.